Monika After Story/game/zz_selector.rpy

```diff
@@ -3940,18 +3940,22 @@ label monika_clothes_select:
             # need to get a list of clothes that have been gifted
             # so we will get a list of all clothes and then remove the event_clothes
             gifted_clothes = mas_selspr.filter_clothes(True)
+            ev_outfits = persistent._mas_event_clothes_map.get(datetime.date.today(), ())
 
             for index in range(len(gifted_clothes)-1, -1, -1):
                 spr_obj = gifted_clothes[index].get_sprobj()
                 if (
-                        not spr_obj.is_custom
-                        and spr_obj != mas_clothes_def
-                        and spr_obj != mas_clothes_blazerless
+                        spr_obj.name in ev_outfits
+                        or (
+                                not spr_obj.is_custom
+                                and spr_obj != mas_clothes_def
+                                and spr_obj != mas_clothes_blazerless
+                        )
                 ):
                     gifted_clothes.pop(index)
 
             #Now we handle holiday clothes
-            clothes_to_add = persistent._mas_event_clothes_map.get(datetime.date.today())
+            clothes_to_add = ev_outfits
```

Comment:
`_mas_event_clothes_map` seems to be a mapping of a date to a single clothing item's name, so there's always just 1 outfit here. In that case, a simple `spr_obj.name == ev_outfits` would be enough, but I also would just use `clothes_to_add` instead (so move the var to before the loop instead of creating an extra temp var).